service/device/registry/test/src/test/resources/features/DeviceRegistry.feature

```diff
@@ -16,237 +16,209 @@ Feature: Device Registry CRUD tests
+        And No exception was thrown
+
+    Scenario: Creating a single device with clientID that contains 256 characters
+        Create a single device with clientID that contains 256 characters.
+        Kapua should return an error.
+
+        Given I expect the exception "KapuaIllegalArgumentException" with the text "Value over than allowed max length. Max length is 255."
+        When I create a device with name "validDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDevicevalidDeviceval"
+        Then An exception was thrown
+
+    Scenario: Creating a single device with clientID that contains invalid characters
+        Create a single device with clientID that contains invalid characters.
+        Kapua should return an error.
+
+        Given I expect the exception "KapuaIllegalArgumentException" with the text "An illegal value was provided for the argument deviceCreator.clientId"
+        When I try to create devices with invalid symbols "#+*&,?>/" in name
+        Then An exception was thrown
+
+    #This scenario must be separated from the above scenario because upper scenario cannot contain (::) characters in a row.
+    Scenario: Creating a single device with clientID that contains invalid character
+        Create a single device with clientID invalid special character ::.
+        Kapua should return an error.
+
+        Given I expect the exception "KapuaIllegalArgumentException" with the text "An illegal value was provided for the argument deviceCreator.clientId"
+        When I create a device with name "testDevice::"
+        Then An exception was thrown
+
+    Scenario: Creating two device with the same clientID
+        Try to create two devices with same clientID.
+        Kapua should return an error.
+
+        Given I expect the exception "KapuaDuplicateNameException" with the text "An entity with the same name testDevice already exists."
+        When I create a device with name "testDevice"
+        And I create a device with name "testDevice"
+        Then An exception was thrown
+
+    Scenario: All device parameters must match the device creator
+        Create a test device and check whether it was created correctly. All the device
+        parameters must match the device creator specifications.
+
+        Given A device named "testDevice"
+        Then The device matches the creator parameters
+
+    Scenario: Case sensitiveness of named device searches
+        Searching by client ID is case sensitive.
+
+        Given A device named "CaseSensitiveTestName"
+        Then Named device registry searches are case sensitive
```

Comment:
Can you please change the name of this step, because it is not clear what it does...?